api/controllers/analytics.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const request = require('request');
+const config = require('../config');
+const common = require('../common');
+
+const LabelNameMatch = {
+    "PhD Student": ["doctoral student", "graduate student","phd candidate","grad student","phd student"],
+    "Faculty" : ["prof", "senior research fellow", "chair", "instructor", "director", "scientist", "advisor", "principal investigator", "pi", "teacher","scholar", "lec"],
+    "Postdoctoral Researcher" : ["postdoctoral" ,"fellow", "research associate", "post", "phd" ,"research"],
+    "Research Assistant" : ["research assistant", "research coordinator", "intern","ra"],
+    "High School Student" : ["high school"],
+    "Clinician" : ["chief physician","clinical researcher","advanced imaging neuroscientist","presurgical tumor evaluation","md","radiologist","neuroradiologist",
+                "logist","physic","clinician","medical doctor","physician","neurologist"],
+    "College Student" : ["under", "teaching assistant",],
+    "Industry" : ["software", "product", "manager", "owner", "developer", "des", "engineer"],
+    "Masters Student" : ["masters", "phil", "mtech", "msc"],
+    "Student (unspecified)" : ["studen"],
+}
+/**
+ * @apiGroup Projects
+ * @api {get} /project/data
+ * @apiDescription              Query data for pie chart
+ * 
+ * @apiHeader {String} [authorization]
+ *                                  A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object (JSON) }             Pie Chart Data
+ */
+ router.get('/data', common.jwt({credentialsRequired: true}), (req, res, next)=> {
+    // if(!req.headers.authorization) next();
+    if(!common.hasScope(req, "admin")) return next("You need to be admin to access this");
+    request.get({ url : config.auth.api+'/profile/list', headers: { authorization: req.headers.authorization}}, (err, _res, json)=> {
```

Comment:
I just remembered that I wanted to deprecate this API on auth service.. It's full of security flag and I think it's best to implement this API on the auth service. So let's move there..

Reply:
Also, "request" is no longer maintained. We should use "axios".